pkg/performanceprofile/controller/performanceprofile/components/kubeletconfig/kubeletconfig_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
-	kubeletv1beta1 "k8s.io/kubernetes/pkg/kubelet/apis/config/v1beta1"
+	eviction "k8s.io/kubernetes/pkg/kubelet/eviction"
```

Comment:
is that a result of this PR or do we need a rebase here ?

Reply:
however I agree it's surprising that the package providing the values changed, did we track down the source of the change?

Reply:
I noted that in the PR description.  
```
kubeletconfig_test.go fix to reflect the latest k8s changes
```